### PR TITLE
Write readme

### PR DESCRIPTION
**Pull request by weswc**
Writing the README.

Fixed bug in link_dependencies in which packages are being checked for in static, development "tmp/rpkgs" instead of dynamic variable link_path

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -15,13 +15,19 @@ knitr::opts_chunk$set(
...
 <!-- badges: end -->
 
-The goal of ghqc is to ...
+The goal of the ghqc ecosystem is to simplify, standardize, and improve traceability of the QC process through the use of shiny apps which create GitHub Issues and Milestones for your project. The ghqc package is a lightweight wrapper to be installed along side project packages and has three primary functionalities:
+
+- **Isolating Package Dependency**: Installs, or symlinks, the high dependency burden of the ghqc shiny app ecosystem to an isolated directory to ensure no interference with any project work.
+
+- **Launching Shiny Apps**: Launches the ghqc shiny apps as background jobs to allow for continued use of the R Console
+
+- **Verifying Configuring Information**: Aids in the installation and structure verification of the organization specific configuring information, such as checklists and logos.
```
Comment (jenna-a2ai) [inline README.Rmd:30]: At the end of this section, we can say something like:
"See the README of ghqc.app for a high-level overview of the core shiny app functionality" and link it

File: README.Rmd [hunk 2]
```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
...
+
+Enter the url of the configuration information repository described above
+
+![](man/figures/renviron_setup.png)
+
+2. **Configuration Information Download and Verification**
+
+The configuration information will default to installing to "~/.local/share/ghqc/<repo_name>". To download to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to download the repository.
+
+In addition to downloading the repository, or remote updates if the repository was previously downloaded, the structure of the repository and the checklist yamls are verified.
```
Comment (jenna-a2ai) [inline README.Rmd]: Maybe we can say something more clearly here about how ghqc will check for remote updates and let the user update the local directory if there any (with a ss)

File: README.Rmd [hunk 3]
```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
...
+
+3. **Shiny App Dependency Installation**
+
+Select whether to install the dependency packages or symlink to previously a previously installed package library. 
+
+- INSTALL PACKAGES
+
+  - NOTE: The install option is currently only available to linux systems supported by Posit Package Manager (PPM) and will install from [PPM@2024-03-01](https://packagemanager.posit.co/cran/2024-03-01). 
+  
+  - The dependency packages are installed by default to "~/.local/share/ghqc/rpkgs". To install to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to install the dependency packages to.
```
Comment (jenna-a2ai) [inline README.Rmd]: Remove "to":
"the path in which to install the dependency packages".

File: README.Rmd [hunk 4]
```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
...
+  
+  - Provide the path to the previously installed package library
+  
+  - The dependency packages are symlinked by default to "~/.local/share/ghqc/rpkgs". To symlink to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to download the repository.
+  
+  ![](man/figures/link_depends.png)
+
+#### ghqc.app installation
+
+You can install the development version of ghqc.app from [GitHub](https://github.com/) with:
```
Comment (jenna-a2ai) [inline README.Rmd:120]: Maybe a special note that the only part missing from the setup is this

**Review by jenna-a2ai — CHANGES_REQUESTED** [review]

File: README.Rmd [hunk 5]
```diff
@@ -15,13 +15,19 @@ knitr::opts_chunk$set(
...
-# ghqc
+# ghqc <a href="https://github.com/a2-ai/ghqc/"><img src="man/figures/logo.png" align="right" height="139" alt="ghqc website" /></a>
 
 <!-- badges: start -->
-[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![R-CMD-check](https://github.com/A2-ai/ghqc/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/A2-ai/ghqc/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-The goal of ghqc is to ...
+The goal of the ghqc ecosystem is to simplify, standardize, and improve traceability of the QC process through the use of shiny apps which create GitHub Issues and Milestones for your project. The ghqc package is a lightweight wrapper to be installed along side project packages and has three primary functionalities:
```
Comment (jenna-a2ai) [inline README.Rmd]: "along side" -> "alongside"

**Review by jenna-a2ai — CHANGES_REQUESTED** [review]

File: README.Rmd [hunk 6]
```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
...
+
+## Configuration Information Repository
+
+The ghqc ecosystem has been developed with the intent organizations can customize elements of the QC process. Currently, the customizable elements are:
+
+- **Checklists**: yaml files contained within a directory named "checklist" *(required)*
+
+- **Prepended Note**: A note to prepend to all checklists named "note" *(optional)*
+
+- **Logo**: A logo to be included in the header of the resultant record's header named as "logo.png"
```
Comment (jenna-a2ai) [inline README.Rmd]: the logo is required, right? add *(required)*

**Review by jenna-a2ai — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 1 above)
Comment (weswc) [reply README.Rmd:30]: https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L129

This has been addressed at the bottom. Reference to the standalone website will be added here once it is live

File: README.Rmd (hunk 2 above)
Comment (weswc) [reply README.Rmd]: https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L86-L92

Changed functionality of `ghqc::setup_ghqc()` to make this more explicit to the user. Also added this section to the README to explicitly discuss update mechanism

File: README.Rmd (hunk 3 above)
Comment (weswc) [reply README.Rmd]: https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L96

Done!

File: README.Rmd (hunk 4 above)
Comment (weswc) [reply README.Rmd:120]: https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L118

Done!

File: README.Rmd (hunk 5 above)
Comment (weswc) [reply README.Rmd]: https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L24

Done!

File: README.Rmd (hunk 6 above)
Comment (weswc) [reply README.Rmd]: https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L55

Done!

**Pull request merged**